linera-views/tests/operations_tests.rs

```diff
@@ -113,7 +116,7 @@ async fn test_readings_vec<OP: KeyValueStoreClient + Sync>(
 
 fn get_random_key_value_vec1(len_value: usize) -> Vec<(Vec<u8>, Vec<u8>)> {
     let key_prefix = vec![0];
-    let n = 1000;
+    let n = 100;
```

Comment:
Is it this a performance issue?

Reply:
Yes, the test test for `n=1000` takes 165s while the test for `n=100` takes 25s.

Reply:
Note that the `DynamoDb` takes 360s for `n=1000`. So, we still have improvement.